dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/trackedentity/query/context/querybuilder/SqlQueryHelper.java

```diff
@@ -139,7 +139,7 @@ static Renderable buildOrderSubQuery(
                       "selectedEventField", "event",
                       "dataElementField", field.render())));
     }
-    if (dimId.isEventDimension() && !isDataElement(dimId)) {
+    if (dimId.isEventDimension()) {
```

Comment:
Is this a bug fix? If so, we have to backport it.

Reply:
it's not a bug, it's just pointless to check if it's not data element since we already checked a few lines above